changes/ticket20218

```diff
@@ -0,0 +1,4 @@
+  o Minor bugfixes (routerstatus_has_changed function):
+    - Check all fields in the function routerstatus_has_changed.
+
+  Fixes bug 20218; bugfix on 0.4.3.x-final
```

Comment:
This might pass `make check-changes`, but it's not how we do changes files:
* there should not be any empty lines between the change and the fix version, and
* the bug fix version is the version that the bug was introduced in, not the current version.

Reply:
You might find these example changes entries helpful:
https://gitweb.torproject.org/tor.git/tree/doc/HACKING/CodingStandards.md#n102

And these instructions for finding the bugfix version:
https://gitweb.torproject.org/tor.git/tree/doc/HACKING/CodingStandards.md#n121

Finding the bugfix version might be hard for this ticket, it's the earliest version of these two versions:
* the version that added "ipv6_orport"
* the version that added "is_v2_dir"